pkg/crd/testdata/Makefile

```diff
@@ -0,0 +1,27 @@
+GEN?=../../../.run-controller-gen.sh output:stdout
+
+all:
+	# Generate separate CRDs for cronjob v1 and v2
+	# - using crd v1
+	$(GEN) crd:crdVersions=v1 paths=./cronjob/v1 > cronjob/v1/cronjob_crdv1.yaml
+	$(GEN) crd:crdVersions=v1 paths=./cronjob/v2 > cronjob/v2/cronjob_crdv1.yaml
+	# - using crd v1beta1
+	$(GEN) crd:crdVersions=v1beta1 paths=./cronjob/v1 > cronjob/v1/cronjob_crdv1beta1.yaml
+	$(GEN) crd:crdVersions=v1beta1 paths=./cronjob/v2 > cronjob/v2/cronjob_crdv1beta1.yaml
+	# Generate a combined CRD of Cronjob v1 & v2
+	# - using crd v1
+	$(GEN) crd:crdVersions=v1 paths=./cronjob/... > cronjob/cronjob_crdv1.yaml
```

Comment:
Whenever we generate yaml manifests covering 2 versions (here `cronjob/v1` and `cronjob/v2`), the same content appears twice in the generated output. In most situations the last content replaces the first one, but this is not the case here when using `output:stdout`.
This should be fixed with https://github.com/kubernetes-sigs/controller-tools/pull/350. These tests at least served to prove that :)